packages/medusa/src/api-v2/store/collections/validators.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  createFindParams,
+  createOperatorMap,
+  createSelectParams,
+} from "../../utils/validators"
+import { z } from "zod"
+
+export const StoreGetCollectionParams = createSelectParams()
+
+export type StoreGetCollectionsParamsType = z.infer<
+  typeof StoreGetCollectionsParams
+>
+export const StoreGetCollectionsParams = createFindParams({
```

Comment:
Once @riqwan merges his PR I'll clean these up to be shared across admin and storefront